bedrock/base/templates/404.html

```diff
@@ -2,47 +2,63 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% set_lang_files "mozorg/404" %}
-
-{% extends "base-pebbles.html" %}
+{% set_lang_files "mozorg/404-update-2020" %}
```

Comment:
Is the plan here to lose translations for the existing pages? Usually when making updates to existing pages we'll use an [L10n conditional tag](https://bedrock.readthedocs.io/en/latest/l10n.html#staging-copy-changes), so locales that don't yet have the new content translated can fall back to the old strings. Technically, we don't _have_ to do that, but it means the error pages would be displayed in English until communities catch up again.

Reply:
It looks like we are wanting to use these pages for testing moving over to fluent. I'm going to get on a bit earlier to talk about this with you and make sure I understand what needs to be done.

Reply:
Happy to chat about it, but don’t worry about the Fluent part. We can tackle that after this PR has merged 🙂